products/solr.md

```diff
@@ -10,6 +10,7 @@ alternate_urls:
 releasePolicyLink: https://solr.apache.org/downloads.html#about-versions-and-support
 changelogTemplate: "https://solr.apache.org/docs/{{'__LATEST__'|replace:'.','_'}}/changes/Changes.html"
 releaseDateColumn: true
+versionCommand: bin/solr version
```

Comment:
Must be moved before the `releasePolicyLink` as per [our style guide](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide).